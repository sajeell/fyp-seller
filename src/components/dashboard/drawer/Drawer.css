:root {
  --background-color: #b9c4cb;
  --font-color: #707070;
  --input-background-color: #efefef;
  --button-background-color: #00978e;
  --button-hover-color: #525151;
}

.drawer-wrapper {
  display: flex;
  flex-direction: column;
  min-height: 100vh;
  height: 100%;
  background-color: var(--background-color);
  width: 250px;
  border-top-right-radius: 20px;
  padding-top: 40px;
  padding-left: 30px;
  border-bottom-right-radius: 20px;
  box-shadow: 3px 5px 10px 0 rgb(0 0 0 / 50%);
}

.drawer-title {
  font-size: 20px;
  font-weight: bold;
}

.drawer-subtitle {
  line-height: 10px;
  color: var(--font-color);
}

.drawer-buttons-wrapper {
  margin-top: 50px;
}

.drawer-button {
  display: flex;
  align-items: center;
  margin-bottom: 16px;
  cursor: pointer;
}

.drawer-button-icon img {
  width: 23px !important;
  margin-right: 17px;
}
