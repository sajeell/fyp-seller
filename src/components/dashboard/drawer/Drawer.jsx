import { logout } from '../../../redux/user/user.action'
import { Link } from 'react-router-dom'
import { useDispatch, useSelector } from 'react-redux'
import { useEffect } from 'react'
import { useNavigate } from 'react-router-dom'

import './Drawer.css'

const Drawer = (props) => {
  let dispatch = useDispatch()
  let navigate = useNavigate()

  const token = useSelector((state) => state.user.token)

  useEffect(() => {
    if (token === null) {
      return navigate('/')
    }
  }, [token])
  return (
    <div className='drawer-wrapper'>
      <div className='drawer-title-row'>
        <div className='drawer-title'>Barganttic</div>
        <div className='drawer-subtitle'>The E-Commerce Company</div>
      </div>
      <div className='drawer-buttons-wrapper'>
        <div className='drawer-button'>
          <div className='drawer-button-text'>
            {props.dashboard === true ? (
              <strong>- Dashboard</strong>
            ) : (
              <Link to='/'>
                <span>- Dashboard</span>
              </Link>
            )}
          </div>
        </div>

        <div className='drawer-button'>
          <div className='drawer-button-text'>
            {props.products === true ? (
              <strong>- Products</strong>
            ) : (
              <Link to='/products'>
                <span>- Products</span>
              </Link>
            )}
          </div>
        </div>

        <div className='drawer-button'>
          <div className='drawer-button-text'>
            {props.orders === true ? (
              <strong>- Orders</strong>
            ) : (
              <Link to='/orders'>
                <span>- Orders</span>
              </Link>
            )}
          </div>
        </div>

        <div className='drawer-button'>
          <div className='drawer-button-text'>
            {props.intermediary === true ? (
              <strong>- Intermediary</strong>
            ) : (
              <Link to='/intermediary'>
                <span>- Intermediary</span>
              </Link>
            )}
          </div>
        </div>

        <div
          className='drawer-button'
          onClick={() => {
            dispatch(logout())
          }}
        >
          <div className='drawer-button-text'>
            <span>- Log Out</span>
          </div>
        </div>
      </div>
    </div>
  )
}

export default Drawer
