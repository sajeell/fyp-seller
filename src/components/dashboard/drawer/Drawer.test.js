import React from 'react'
import renderer from 'react-test-renderer'

import Drawer from './Drawer'

it('renders correctly when there is no props', () => {
  const tree = renderer.create(<Drawer />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('makes the dashboard text bold when dashboard in the props is true', () => {
  const tree = renderer.create(<Drawer dashboard={true} />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('makes the driver text bold when driver in the props is true', () => {
  const tree = renderer.create(<Drawer driver={true} />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('makes the parents text bold when parents in the props is true', () => {
  const tree = renderer.create(<Drawer parents={true} />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('makes the students text bold when students in the props is true', () => {
  const tree = renderer.create(<Drawer students={true} />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('makes the trip text bold when trip in the props is true', () => {
  const tree = renderer.create(<Drawer trip={true} />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('makes the bus text bold when bus in the props is true', () => {
  const tree = renderer.create(<Drawer bus={true} />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('makes the settings text bold when settings in the props is true', () => {
  const tree = renderer.create(<Drawer settings={true} />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('makes the help text bold when help in the props is true', () => {
  const tree = renderer.create(<Drawer help={true} />).toJSON()
  expect(tree).toMatchSnapshot()
})
