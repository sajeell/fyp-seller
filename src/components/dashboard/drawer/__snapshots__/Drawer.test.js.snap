// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`makes the bus text bold when bus in the props is true 1`] = `
<div
  className="drawer-wrapper"
>
  <div
    className="drawer-title-row"
  >
    <div
      className="drawer-title"
    >
      SchooLync
    </div>
    <div
      className="drawer-subtitle"
    >
      The Transport Company
    </div>
  </div>
  <div
    className="drawer-buttons-wrapper"
  >
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="dashboard.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Dashboard
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="driver.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Driver
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="bus.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <strong>
          Bus
        </strong>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="trip.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Trip
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="parent.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Parents
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="student.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Students
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="settings.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Settings
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="help.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Help
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`makes the dashboard text bold when dashboard in the props is true 1`] = `
<div
  className="drawer-wrapper"
>
  <div
    className="drawer-title-row"
  >
    <div
      className="drawer-title"
    >
      SchooLync
    </div>
    <div
      className="drawer-subtitle"
    >
      The Transport Company
    </div>
  </div>
  <div
    className="drawer-buttons-wrapper"
  >
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="dashboard.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <strong>
          Dashboard
        </strong>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="driver.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Driver
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="bus.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Bus
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="trip.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Trip
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="parent.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Parents
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="student.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Students
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="settings.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Settings
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="help.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Help
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`makes the driver text bold when driver in the props is true 1`] = `
<div
  className="drawer-wrapper"
>
  <div
    className="drawer-title-row"
  >
    <div
      className="drawer-title"
    >
      SchooLync
    </div>
    <div
      className="drawer-subtitle"
    >
      The Transport Company
    </div>
  </div>
  <div
    className="drawer-buttons-wrapper"
  >
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="dashboard.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Dashboard
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="driver.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <strong>
          Driver
        </strong>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="bus.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Bus
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="trip.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Trip
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="parent.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Parents
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="student.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Students
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="settings.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Settings
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="help.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Help
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`makes the help text bold when help in the props is true 1`] = `
<div
  className="drawer-wrapper"
>
  <div
    className="drawer-title-row"
  >
    <div
      className="drawer-title"
    >
      SchooLync
    </div>
    <div
      className="drawer-subtitle"
    >
      The Transport Company
    </div>
  </div>
  <div
    className="drawer-buttons-wrapper"
  >
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="dashboard.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Dashboard
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="driver.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Driver
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="bus.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Bus
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="trip.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Trip
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="parent.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Parents
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="student.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Students
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="settings.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Settings
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="help.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <strong>
          Help
        </strong>
      </div>
    </div>
  </div>
</div>
`;

exports[`makes the parents text bold when parents in the props is true 1`] = `
<div
  className="drawer-wrapper"
>
  <div
    className="drawer-title-row"
  >
    <div
      className="drawer-title"
    >
      SchooLync
    </div>
    <div
      className="drawer-subtitle"
    >
      The Transport Company
    </div>
  </div>
  <div
    className="drawer-buttons-wrapper"
  >
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="dashboard.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Dashboard
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="driver.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Driver
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="bus.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Bus
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="trip.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Trip
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="parent.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <strong>
          Parents
        </strong>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="student.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Students
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="settings.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Settings
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="help.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Help
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`makes the settings text bold when settings in the props is true 1`] = `
<div
  className="drawer-wrapper"
>
  <div
    className="drawer-title-row"
  >
    <div
      className="drawer-title"
    >
      SchooLync
    </div>
    <div
      className="drawer-subtitle"
    >
      The Transport Company
    </div>
  </div>
  <div
    className="drawer-buttons-wrapper"
  >
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="dashboard.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Dashboard
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="driver.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Driver
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="bus.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Bus
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="trip.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Trip
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="parent.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Parents
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="student.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Students
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="settings.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <strong>
          Settings
        </strong>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="help.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Help
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`makes the students text bold when students in the props is true 1`] = `
<div
  className="drawer-wrapper"
>
  <div
    className="drawer-title-row"
  >
    <div
      className="drawer-title"
    >
      SchooLync
    </div>
    <div
      className="drawer-subtitle"
    >
      The Transport Company
    </div>
  </div>
  <div
    className="drawer-buttons-wrapper"
  >
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="dashboard.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Dashboard
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="driver.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Driver
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="bus.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Bus
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="trip.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Trip
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="parent.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Parents
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="student.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <strong>
          Students
        </strong>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="settings.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Settings
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="help.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Help
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`makes the trip text bold when trip in the props is true 1`] = `
<div
  className="drawer-wrapper"
>
  <div
    className="drawer-title-row"
  >
    <div
      className="drawer-title"
    >
      SchooLync
    </div>
    <div
      className="drawer-subtitle"
    >
      The Transport Company
    </div>
  </div>
  <div
    className="drawer-buttons-wrapper"
  >
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="dashboard.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Dashboard
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="driver.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Driver
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="bus.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Bus
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="trip.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <strong>
          Trip
        </strong>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="parent.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Parents
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="student.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Students
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="settings.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Settings
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="help.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Help
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders correctly when there is no props 1`] = `
<div
  className="drawer-wrapper"
>
  <div
    className="drawer-title-row"
  >
    <div
      className="drawer-title"
    >
      SchooLync
    </div>
    <div
      className="drawer-subtitle"
    >
      The Transport Company
    </div>
  </div>
  <div
    className="drawer-buttons-wrapper"
  >
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="dashboard.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Dashboard
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="driver.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Driver
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="bus.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Bus
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="trip.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Trip
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="parent.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Parents
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="student.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Students
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="settings.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Settings
        </span>
      </div>
    </div>
    <div
      className="drawer-button"
    >
      <div
        className="drawer-button-icon"
      >
        <img
          alt="Home icon"
          src="help.png"
        />
      </div>
      <div
        className="drawer-button-text"
      >
        <span>
          Help
        </span>
      </div>
    </div>
  </div>
</div>
`;
